CHANGE.txt

```diff
@@ -2,6 +2,13 @@
 LabKey Python Client API News
 +++++++++++
 
+What's New in the LabKey 3.1.0 package
+==============================
+
+*Release date: TBD*
```

Comment:
@labkey-alan do you want me to set this to something before the PR gets merged?

Reply:
I don't believe 3.0.0 has been released yet so it may be that these just get put in under that tag. Alan will confirm.

Reply:
Correct, this will be part of 3.0.0, so just put your release notes under that version.

Reply:
Great. Done